cfbs.json

```diff
@@ -180,6 +180,20 @@
         "copy ./disable_prelinking.cf services/autorun/disable_prelinking.cf"
       ]
     },
+    "enable-aslr": {
+      "description": "Ensure that Address space layout randomization (aslr) is enabled on the system",
+      "tags": ["supported", "security", "compliance"],
+      "repo": "https://github.com/cfengine/modules",
+      "by": "https://github.com/craigcomstock",
+      "version": "0.0.1",
+      "commit": "d68c4143d242e631743fae2f6f7620ea882c4d42",
```

Comment:
will update this commit when https://github.com/cfengine/modules/pull/57 is merged